src/main/java/Calculator/Calculator.java

```diff
@@ -0,0 +1,118 @@
+package Calculator;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public final class Calculator {
+    private final String customSplitterStarter = "//";
+    private final String customSplitterEnder = "\\n";
+    private final String[] defaultSplitters = {",", ";"};
+    private final int defaultReturnValue = 0;
+    private final String regexSpecialChars = ".^$*+?{}[]()|\\";
+
+    public int calculate(String input) {
+        if (isNullOrBlank(input)) return defaultReturnValue;
+        List<Integer> numbers = parseNumbers(input);
+        return calculateSum(numbers);
+    }
+
+    private Boolean isNullOrBlank(String input) {
```

Comment:
wrapper 타입을 사용하신 이유가 있으실까요?
```suggestion
    private boolean isNullOrBlank(String input) {
```